ext/src/http/client/curl/http_operation_curl.cc

```diff
@@ -8,6 +8,11 @@
 #include "opentelemetry/ext/http/client/curl/http_client_curl.h"
 #include "opentelemetry/sdk/common/global_log_handler.h"
 
+// CURL_VERSION_BITS was added in CURL 7.43.0
```

Comment:
Check the version of CURL before define CURL_VERSION_BITS?

Reply:
The code uses ifndef, and only adds the macro if it is not provided already by the curl headers included earlier.

I think this is the most robust form, testing the curl version is more error prone.